jvb/src/main/kotlin/org/jitsi/videobridge/stats/StatsManager.kt

```diff
@@ -151,11 +143,19 @@ class StatsManager(
     companion object {
         @JvmField
         val config = StatsManagerConfig()
-    }
 
-    init {
-        val period = config.interval.toMillis()
-        require(period >= 1) { "period $period" }
-        statisticsRunnable = StatisticsPeriodicRunnable(statistics, period)
+        /**
+         * The [RecurringRunnableExecutor] which periodically invokes [statisticsRunnable].
+         */
+        private val statisticsExecutor = RecurringRunnableExecutor(
```

Comment:
We'll only have a single instance of `StatsManager` anyway, right?

Reply:
Yes

Reply:
I'd leave them as members (we'd also want a hook in ctor to override them if we want to test this)